starlite/storage/base.py

```diff
@@ -86,21 +85,21 @@ def new(cls, data: bytes, expires_in: int | timedelta | None) -> StorageObject:
             expires_in = timedelta(seconds=expires_in)
         return cls(
             data=data,
-            expires_at=(datetime.now() + expires_in) if expires_in else None,
+            expires_at=(datetime.now(tz=timezone.utc) + expires_in) if expires_in else None,
         )
 
     @property
     def expired(self) -> bool:
         """Return if the :class:`StorageObject` is expired"""
-        return self.expires_at is not None and datetime.now() >= self.expires_at
+        return self.expires_at is not None and datetime.now(tz=timezone.utc) >= self.expires_at
 
     @property
-    def expires_in(self) -> int:
+    def expires_in(self) -> float | int:
         """Return the expiry time of this ``StorageObject`` in seconds. If no expiry time
         was set, return ``-1``.
         """
         if self.expires_at:
-            return (self.expires_at - datetime.now()).seconds
+            return (self.expires_at - datetime.now(tz=timezone.utc)).total_seconds()
```

Comment:
```suggestion
            return (self.expires_at - datetime.now(tz=timezone.utc)).seconds
```

Seconds is correct here since ttl is expressed in seconds for the other backends as well (e.g. coming from redis)